tests/network/services.go

```diff
@@ -103,7 +106,7 @@ var _ = SIGDescribe("[Serial]Services", func() {
 		serviceFQDN := fmt.Sprintf("%s.%s", serviceName, namespace)
 
 		By(fmt.Sprintf("starting a job which tries to reach the vmi via service %s", serviceFQDN))
-		job := runTCPClientExpectingHelloWorldFromServer(serviceFQDN, strconv.Itoa(servicePort), namespace, isIPv6)
+		job := runTCPClientExpectingHelloWorldFromServer(serviceFQDN, strconv.Itoa(servicePort), namespace, isIPv6, 0)
```

Comment:
Are we expecting that the connectivity goes away instantly?

Reply:
If not the job is not going to be marked as Failed

Reply:
I see it now.
just a note: It looks like we use it in 2 same tests under different contexts (but the context doesn't change the test case).